cli/cmd/summary.go

```diff
@@ -62,3 +73,87 @@ func checkFilePath(filename string) error {
 	dirPath := filepath.Dir(filename)
 	return os.MkdirAll(dirPath, 0755)
 }
+
+func generateSummaryTable() *schema.Table {
+	tableName := "cloudquery_sync_summaries"
+	t := schema.Tables{{
+		Name: tableName,
+		Transform: transformers.TransformWithStruct(
+			&syncSummary{},
+		),
+		Columns: []schema.Column{},
+	}}
+	if err := transformers.TransformTables(t); err != nil {
+		panic(err)
+	}
+	return t[0]
+}
+
+func sendSummary(writeClients []plugin.Plugin_WriteClient, destinationSpecs []specs.Destination, destinationsClients []*managedplugin.Client, destinationTransformers []*transformer.RecordTransformer, summary *syncSummary, noMigrate bool) error {
+	summaryTable := generateSummaryTable()
+	summaryTableSchema := summaryTable.ToArrowSchema()
+
+	defaultCaser := caser.New()
+	for i := range destinationsClients {
+		if !destinationSpecs[i].SyncSummary {
+			continue
+		}
+
+		transformedSchema := destinationTransformers[i].TransformSchema(summaryTableSchema)
+		transformedSchemaBytes, err := plugin.SchemaToBytes(transformedSchema)
+		if err != nil {
+			return err
+		}
+		wr := &plugin.Write_Request{}
+
+		// Respect the noMigrate flag
+		if !noMigrate {
```

Comment:
Since we support `noMigrate` shouldn't we also call `generateSummaryTable` inside `migrate_v3.go`?

Reply:
Done in b12c0d9